### ISSUE TITLE
Copy to clipboard in Interactive Mode not work

### ISSUE THREAD

**Issue by manhcuong0312** [Andre9642/nvda-markdownForever#13]
The title says all. I open the Interactive Mode dialog, set everything fixed my need, press copy to clipboard button, and press NVDA+C. NVDA says: no text in clipboard.  
Using Windows 10, NVDA 2020.3  
Cuong

**Comment by AAClause** [comment]
Hi,
Thanks for the report.
Is it about formatted HTML?
Are you able to paste into Microsoft Word or rich text area field anyway?
If the answer is yes for the previous questions, it's not a bug. The copy as a Formatted HTML looks like a file, so NVDA cannot read the clipboard content. I'll try to find another implementation.
Please keep the issue open for reference.

**Comment by manhcuong0312** [comment]
Hi @Andre9642  
Is it about formatted HTML? No. I converted to HTML  
Are you able to paste into Microsoft Word or rich text area field anyway? I'm able to paste in Microsoft Word, and it presents as formatted html  
To be clerify, I'm so lazy so each time I have something need to be updated from an md file, I prefer converting that file to html source, put it to clipboard then paste into old html file I converted before. Doing in interactive mode of Markdown Forever, NVDA didn't read the text, and I can't paste that text in to the old html file so i think this is a bug. Please tell if I misunderstand something.  
Thank you.

**Comment by tmthywynn8** [comment]
I think you misunderstood something. The Convert to: combo-box  should be set to HTML source not just HTML which is the default.
